numba/tests/test_typedlist.py

```diff
@@ -155,6 +155,19 @@ def test_unsigned_access(self):
         self.assertEqual(L.pop(ui32_1), 2)
         self.assertEqual(L.pop(ui32_0), 123)
 
+    def test_dtype(self):
+        L = List()
+        self.assertEqual(L.dtype, None)
+
+        L = List.empty_list(int32)
+        self.assertEqual(L.dtype, int32)
+
+        L = List.empty_list(float32)
+        self.assertEqual(L.dtype, float32)
+
+        L = List.empty_list(unicode_type)
+        self.assertEqual(L.dtype, unicode_type)
```

Comment:
The public CI seems to be failing, I think there might be an import missing here.

Reply:
I removed this test